src/modules/common/components/Page.js

```diff
@@ -3,18 +3,18 @@ import PropTypes from 'prop-types'
 
 import Caption from 'modules/common/components/Caption'
 import RemoteContent from 'modules/common/components/RemoteContent'
-import CategoryModel from '../../../modules/endpoint/models/CategoryModel'
 
 class Page extends React.Component {
```

Comment:
Is the name Page here still correct? Or should it be Category?

Reply:
yeah it is still correct, because the disclaimerPage also uses this, so this is also the reason why I moved it to common